### ISSUE TITLE
WCAG 3 for personal and small business websites

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/silver#390]
This issue is per request of @jspellman, as discussed on today's Silver call. The numbers in this post aren't real costs of any particular organisation, they're just examples.

Testing a site for WCAG is labor intensive. Depending on a variety of factors, it often takes at least a full day of work, often quite a bit more. For an organization of a budget of 100k doing an annual accessibility audit isn't a big deal. But if you have 8k to build a site, it is hard to justify spending 2k of it on an accessibility audit. That's not even fixing any accessibility problems, it's just to test for them.

The need for accessibility is universal. So WCAG 3 should be too. In my opinion the lowest conformance level of WCAG 3 should be one that is achievable by low-budget and no-budget websites. Ideally it is one that can be tested with minimal effort, without the involvement of an expert.

Having a lower entry level to WCAG 3, makes WCAG more inclusive. It may work better in countries where there is no legal presidency too, and it gives a starting point for organizations new to accessibility.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
